src_c/simd_blitters_sse2.c

```diff
@@ -788,14 +612,15 @@ alphablit_alpha_sse2_argb_no_surf_alpha_opaque_dst(SDL_BlitInfo *info)
     }
 }
 
-#if defined(ENV64BIT)
 void
 blit_blend_rgba_mul_sse2(SDL_BlitInfo *info)
 {
     int n;
     int width = info->width;
     int height = info->height;
 
+    printf("inside rgba mul sse\n");
```

Comment:
I guess this still needs to get removed.

Reply:
Yes, thank you. 😄